### PR TITLE
Removes leftover docs about ansible installer

### LINKED ISSUE #3908: Remove pulp_installer leftover docs

### PR DESCRIPTION
**Pull request by bmbouter**
closes #3908

### CODE REVIEW

**Review by decko — APPROVED** [review]
LGTM

**Review by mikedep333 — APPROVED** [review]
I would like to move the template file into the docs, but this is good enough for now.

Thank you!

File: docs/installation/instructions.rst [hunk 1]
```diff
@@ -224,65 +224,41 @@ You then need to add redis to your :ref:`configuration <configuration>`, such as
...
-   setting the variables according to the `pulpcore_content config variables documentation <https://
-   github.com/pulp/ pulp_installer/tree/master/roles/pulp_content#role-variables>`_
+   content to clients. We recommend adapting with the `pulpcore-content template <https://github.com
+   /pulp/pulp_installer/blob/master/roles/pulp_content/templates/pulpcore-content.service.j2>`_.
 
-2. Make a ``pulpcore-api.service`` file for the pulpcore-api service which serves the Pulp REST API. We
-   recommend starting with the `pulpcore-api template <https://github.com/pulp/pulp_installer/blob/master/roles/pulp_api/templates/pulpcore-api.service.j2>`_
-   and setting the variables according to the `pulpcore-api config variables documentation <https://github.com/pulp/pulp_installer/tree/master/roles/pulp_api#role-variables>`_
+2. Make a ``pulpcore-api.service`` file for the pulpcore-api service which serves the Pulp REST API.
+   We recommend adapting the `pulpcore-api template <https://github.com/pulp/pulp_installer/
```
[the remark this answers is not in the document]
Comment (decko) [reply docs/installation/instructions.rst:239]: I agree with @dkliban but also think that those examples could come in another PR.

[the remark this answers is not in the document]
Comment (bmbouter) [reply docs/installation/instructions.rst:239]: +1 I can update these. Do you know where in the pulp-oci-images repo one would be?

[the remark this answers is not in the document]
Comment (dkliban) [reply docs/installation/instructions.rst:239]: All we have is an issue saying we should add some docs that link to https://www.redhat.com/sysadmin/quadlet-podman.

I would say just link to that article. 

https://github.com/pulp/pulp-oci-images/issues/438

[the remark this answers is not in the document]
Comment (bmbouter) [reply docs/installation/instructions.rst:239]: Without ready-to-go examples I'm going to leave these links here as is.

**Pull request merged**